app/assets/javascripts/modules/sticky-element-container.js

```diff
@@ -7,111 +7,110 @@
   Use 'data-module="sticky-element-container"' to instantiate, and add
   `[data-sticky-element]` to the child you want to position.
 */
+
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
 (function (Modules) {
-  'use strict'
+  function StickyElementContainer (element) {
+    this.wrapper = element
+    this.stickyElement = this.wrapper.querySelector('[data-sticky-element]')
```

Comment:
Maybe out of scope for this PR, but I think it might be worth putting a check for the existence of this element before proceeding in the `init` function e.g. `if (!this.stickyElement) return`

Reply:
Added great spot - thanks Andy